examples/energy-management-app/energy-management-common/src/EVSEManufacturerImpl.cpp

```diff
@@ -232,8 +232,8 @@ void EVSEManufacturer::FakeReadingsUpdate()
 
     // Update meter values
     // Avoid using floats - so we will do a basic rand() call which will generate a integer value between 0 and RAND_MAX
-    // first compute power as a mean + some random value in range 0 to mPowerRandomness_mW
-    int64_t power = (rand() % gFakeReadingsData.mPowerRandomness_mW);
+    // first compute power as a mean + some random value in range +/- mPowerRandomness_mW
+    int64_t power = ((int64_t) rand() % (2 * gFakeReadingsData.mPowerRandomness_mW)) - gFakeReadingsData.mPowerRandomness_mW;
```

Comment:
please use C++ style cast - static_cast<int64_t>(rand()). Not sure if mPowerRandomness is already a 64

Reply:
Changed